### DISCUSSION TITLE
Vits batch size

### DISCUSSION THREAD

**Discussion by KyuubiYoru** [coqui-ai/TTS#1824] [General Q&A] [answered]
What effect has the batch size on the training result?
Does it effect the quality of the model or only the training speed?

**Comment by harmlessman** [comment, marked as answer]
Both the quality and speed of the model affect.

If the batch size is too large, there is a risk of overfitting, and if the batch size is too small, there is a risk of noise.

If the batch size is large, the speed is slow, and if the batch size is small, the speed is fast.

The batch size is recommended between 32 and 64.
